ui/next.config.ts

```diff
@@ -1,7 +1,56 @@
 import type { NextConfig } from "next";
+import path from 'path';
+import fs from 'fs';
+
+// Get the extension name dynamically from the directory structure
+const getExtensionName = () => {
+  try {
+    // Get the current directory
+    const currentDir = __dirname;
+    // Extract the extension name from the path (assuming format: custom_nodes/extension_name/ui)
+    const extensionName = path.basename(path.dirname(currentDir));
+    return extensionName;
+  } catch (error) {
+    console.warn('Failed to get extension name dynamically:', error);
+    // Fallback to the current name
+    return 'ComfyStream';
+  }
+};
+
+const extensionName = getExtensionName();
+
+// Check if we're in development mode
+// This is set by the NEXT_PUBLIC_DEV environment variable in package.json scripts
+const isDev = process.env.NEXT_PUBLIC_DEV === 'true';
```

Comment:
@eliteprox, @ryanontheinside Could we explore alternative ways to integrate the website into the node? Th
This version improves clarity, flow, and readability while keeping it professional and concise. Let me know if you'd like any further refinements!

Reply:
I think this will be solved by using https://github.com/yondonfu/comfystream/pull/98 right?

Reply:
I agree, I've included `npm run build` in the devcontainer startup here so these files are generated for developers. 
https://github.com/yondonfu/comfystream/blob/f50c546ddb252eafa944a45e3b5218831bb8a54b/.devcontainer/post-create.sh#L14

[install.py](https://github.com/yondonfu/comfystream/blob/f50c546ddb252eafa944a45e3b5218831bb8a54b/install.py) will download them automatically using a version tag

Reply:
I agree, I've included `npm run build` in the devcontainer startup here so these files are generated for developers. 
https://github.com/yondonfu/comfystream/blob/f50c546ddb252eafa944a45e3b5218831bb8a54b/.devcontainer/post-create.sh#L14

install.py https://github.com/yondonfu/comfystream/blob/f50c546ddb252eafa944a45e3b5218831bb8a54b/install.py#L105-L106 will download the front-end assets automatically using a version tag